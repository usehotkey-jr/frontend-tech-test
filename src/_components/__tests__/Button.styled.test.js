import {createTestSnapshot} from "../../_helpers/jest/createTestSnapshot.ignore";
import {Button} from "../Button.styled";

describe("Button", () => {
    test("should correctly render", () => {
        createTestSnapshot.shallow(Button);
    });
});
