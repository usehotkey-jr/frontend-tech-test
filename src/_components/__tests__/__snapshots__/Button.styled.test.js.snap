// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should correctly render 1`] = `
<button
  className="sc-bdVaJa gHojya"
/>
`;
